pkg/config/config.go

```diff
@@ -26,11 +26,12 @@ import (
 
 // Out of box defaults
 const (
-	COLLECTOR_API_VERSION      = "2.4.0"
+	COLLECTOR_API_VERSION      = "2.5.0"
```

Comment:
Should this be updated?

Reply:
I believe it should remain 2.4, in release 2.4.

Reply:
nope, will change that back to 2.4. thanks!